recipes/gromacs/2024/gh200-mpi/config.yaml

```diff
@@ -0,0 +1,7 @@
+name: gromacs-gh200
+store: /user-environment
+description: GPU-optimised Gromacs with and without Plumed, and the toolchain to build your own Gromacs.
```

Comment:
there is no plumed version in this recipe.
I assume that we will add a plumed version?